06-multiple-regression.Rmd

```diff
@@ -1099,7 +1099,129 @@ Observe how the regression table for the interaction model has 2 more rows (6 ve
+
+```{r}
+get_regression_points(model_2_interaction) %>% 
+  summarize(var_y = var(average_sat_math), var_y_hat = var(average_sat_math_hat), var_residual = var(residual))
+```
+
+Observe that the variance of $y$ is equal to the variance of $\widehat{y}$ plus the variance of the residuals. But what do these three terms tell us individually?
+
+First, the variance of $y$ (denote this $var(y)$) tells us how much do Massachusetts high schools differ in average math SAT scores. The goal of regression modeling is to fit a model that hopefully *explains* this variation. In other words, what factors explain why certain schools have high math SAT scores, while others have low scores. This is independent of the model; this is just data. In other words, whether we fit an interaction or parallel slopes model, $var(y)$ remains the same. 
+
+Second, the variance of $\widehat{y}$ (denote this $var(\widehat{y})$) tells us how much the fitted values from our interaction model vary. In other words, after account for the percentage of the study body that is socieconomically disadvantaged and school size in an interaction model, how much do our model's explanations of average math SAT scores vary?
+
+Third, the variance of the residuals tells us how much do "the left-overs" from the model vary. Observe how the points in the left-hand plot of Figure \@ref(fig:numxcatx-comparison-2) scatter around the three lines. Say instead all the points fell *exactly* on one of the three lines. Then all residuals would be zero and hence the variance of the residuals would be zero. 
+
+We're now ready to introduce $R^2$:
+
+$$
+R^2 = \frac{var(\widehat{y})}{var(y)}
+$$
+
+It is *the proportion of the spread/variation of the outcome variable $y$ that is explained by our model*, where our model's explanatory power is embedded in the fitted values $\widehat{y}$. Furthermore, since it can be mathematically proven that $0 \leq var(\widehat{y}) \leq var(y)$ (a fact we leave for an advanced class on regression), we are guaranteed that:
+
+$$
+0 \leq R^2 \leq 1
+$$
+
+$R^2$ can be interpreted as follows:
+
+1. $R^2$ values of 0 tell us that our model explains 0% of the variation in $y$. In other words, say we fit a model to the Massachusetts high school data and obtained $R^2 = 0$. This would be telling us that the combination of explanatory variables $x$ we used and model form we chose (interaction or parallel slopes) tell us *nothing* about average math SAT scores. In other words, the model is poor fit.
+1. $R^2$ values of 1 tell us that our model explains 100% of the variation in $y$. In other words, say we fit a model to the Massachusetts high school data and obtained $R^2 = 1$. This would be telling us that the combination of explanatory variables $x$ we used and model form we chose (interaction or parallel slopes) tell us *everything we need to know* about average math SAT scores. 
```

Comment:
```suggestion
1. $R^2$ values of 1 tell us that our model explains 100% of the variation in $y$. Say we fit a model to the Massachusetts high school data and obtained $R^2 = 1$. This would be telling us that the combination of explanatory variables $x$ we used and model form we chose (interaction or parallel slopes) tell us *everything we need to know* about average math SAT scores. 
```